purefb/README.md

```diff
@@ -0,0 +1,139 @@
+# Agent Check: PureFB
+
+## Overview
+
+This check monitors the [Pure Storage FlashBlade][3] through the [Datadog Agent][2] and the [Pure Storage FlashBlade OpenMetrics exporter][1]. 
```

Comment:
Would it make more sense to link directly to the FlashBlade page in the docs for [3]? [https://www.purestorage.com/products/file-and-object/flashblade.html](https://www.purestorage.com/products/file-and-object/flashblade.html)